### PR TITLE
Make sure that "controls" takes the correct value

### PR DESCRIPTION
**Pull request by dugajean**

### CODE REVIEW

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ class Video {
 	 *
 	 * @var string
 	 */
-	const PLAYER_VER = '1.3.3';
+	const PLAYER_VER = '1.4.0';
```
Comment (pereirinha) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php:55]: @dugajean 

Can we test upgrading also the CORE_VER to `2.6.3` — https://github.com/cloudinary/pkg-cloudinary-core?

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php [hunk 2]
```diff
@@ -348,7 +348,6 @@ public function print_video_scripts() {
 				$default       = array(
 					'publicId'    => $cloudinary_id,
 					'sourceTypes' => array( $video['format'] ), // @todo Make this based on eager items as mentioned above.
-					'controls'    => 'on' === $this->config['video_controls'] ? true : false,
```
Comment (pereirinha) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php:351]: @dugajean 

Do we want to remove the `controls` key from the default array?

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php [hunk 3]
```diff
@@ -357,15 +356,15 @@ public function print_video_scripts() {
 				if ( $default['autoplay'] && in_array( $this->config['video_autoplay_mode'], $valid_autoplay_modes, true ) ) {
 					$default['autoplayMode'] = $this->config['video_autoplay_mode'];
 				}
-
+
```
Comment (pereirinha) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php:359]: @dugajean 
can we remove the tabs/spaces added here?

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php [hunk 4]
```diff
@@ -357,15 +356,15 @@ public function print_video_scripts() {
 				if ( $default['autoplay'] && in_array( $this->config['video_autoplay_mode'], $valid_autoplay_modes, true ) ) {
 					$default['autoplayMode'] = $this->config['video_autoplay_mode'];
 				}
-
+				
 				$config = wp_parse_args( $video['args'], $default );
-
+
```
Comment (pereirinha) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php:361]: Same

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php (hunk 2 above)
Comment (dugajean) [reply cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php:351]: It's being added below under all circumstances, so no need for duplicate code :)

**Pull request merged**